lms/djangoapps/mobile_api/video_outlines/serializers.py

```diff
@@ -179,6 +179,20 @@ def video_summary(course, course_id, video_descriptor, request, local_cache):
     """
     returns summary dict for the given video module
     """
+    if video_descriptor.only_on_web:
+        return {
+            "video_url": None,
+            "video_thumbnail_url": None,
+            "duration": 0,
+            "size": 0,
+            "name": video_descriptor.display_name,
+            "transcripts": {},
+            "language": None,
+            "category": video_descriptor.category,
+            "only_on_web": True,
+            "id": unicode(video_descriptor.scope_ids.usage_id),
+        }
```

Comment:
I'm a bit nervous about this change because a developer may just add a new field to the common dict. That said, I will make the change so we can see what that looks like.